atm.js

```diff
@@ -1,3 +1,88 @@
+var checkingInput;
+var checkingBalance ;
+var currentBalance;
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+  checkingInput = $("#checking > .input");
+  checkingBalance = $("#checking > .balance");
+  currentBalance = $("checkingBalance" + "checkingInput")
+  console.log("hello");
+  $(".deposit:first").on("click", function(evt){
+    evt.preventDefault();
+    var currentChecking = parseFloat($('.balance:eq(0)').html().split('$')[1])
+
+    $(".balance:eq(0)").html("$" + (parseFloat(checkingInput.val())+currentChecking).toFixed(2));
+    currentChecking = parseFloat($('.balance:eq(0)').html().split('$')[1])
+
+    console.log(currentChecking)
+    if(currentChecking == 0) {
+      $("#checking").addClass("zero")
+    } else {
+      $("#checking").removeClass("zero")
+    };
+  });
+
+  $(".withdraw:first").on("click", function(evt){
+    evt.preventDefault();
+    var checkingIn = checkingInput.val()
+    console.log(checkingIn)
+    var currentChecking = parseFloat($('.balance:eq(0)').html().split('$')[1])
+    if (checkingIn <= currentChecking) {
+      $(".balance:eq(0)").html("$" + (parseFloat(currentChecking - checkingInput.val())).toFixed(2));
+    }
+    currentChecking = parseFloat($('.balance:eq(0)').html().split('$')[1])
+
+    if(currentChecking == 0) {
+      $("#checking").addClass("zero")
+    } else {
+      $("#checking").removeClass("zero")
+    };
+  });
+  
+
+  var savingsInput = $("#savings > .input");
+  var savingsBalance = $("#savings > .balance");
+  var currentSavingsBalance = $("savingsBalance" + "savingsInput")
+  $(".deposit:eq(1)").on("click", function(evt){
+    evt.preventDefault();
+    var currentSavings = parseFloat($('.balance:eq(1)').html().split('$')[1])
+    $(".balance:eq(1)").html("$" + (parseFloat(savingsInput.val())+currentSavings).toFixed(2));
+    currentSavings = parseFloat($('.balance:eq(1)').html().split('$')[1])
+
+    console.log(currentSavings)
+    if(currentSavings == 0) {
+      $("#savings").addClass("zero")
+    } else {
+      $("#savings").removeClass("zero")
+    };
+  });
+
+  $(".withdraw:eq(1)").on("click", function(evt){
+    evt.preventDefault();
+
+    var savingsIn = savingsInput.val()
+    console.log(savingsIn)
+    var currentSavings = parseFloat($('.balance:eq(1)').html().split('$')[1])
+    if (savingsIn <= currentSavings) {
+      $(".balance:eq(1)").html("$" + (parseFloat(currentSavings - savingsInput.val())).toFixed(2));
+    }
+    currentSavings = parseFloat($('.balance:eq(0)').html().split('$')[1])
+
+    if(currentSavings == 0) {
+      $("#savings").addClass("zero")
+    } else {
+      $("#savings").removeClass("zero")
+    };
+  });
 });
+
```

Comment:
looks good and runs exactly as specified overall. I would review the parts of the assignment you had trouble with on your own, thinking about what you didn't know at the time and had to get help with, just to get extra repetitions on things tentatively understood. Good work Ahmed!